src/photos/styles/topbar.styl

```diff
@@ -31,7 +31,7 @@ $coz-bar-size = 3rem
     background   embedurl('../assets/icons/icon-back.svg')
     cursor       pointer
 
-    +tiny-screen()
+    +small-screen()
```

Comment:
This one is just a random fix of a bug I stumbled upon. The return arrow was messed up between mobile & desktop viewport. This fixes it.